src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/client/UserProfileRequestHandlerTest.java

```diff
@@ -176,7 +173,7 @@ public <T> T sendDeleteWithoutBody(MockMvc mockMvc,
         MvcResult result = mockMvc.perform(delete(path)
                 .headers(getMultipleAuthHeaders())
                 .contentType(APPLICATION_JSON))
-                .andExpect(status().is(expectedHttpStatus.value())).andReturn();
```

Comment:
any reason for removing this? Can you point the code where you are really asserting the http status value?